jOOQ-meta-extensions/src/main/java/org/jooq/meta/extensions/liquibase/LiquibaseDatabase.java

```diff
@@ -97,7 +97,8 @@
     @Override
     protected void export() throws Exception {
         String scripts = getProperties().getProperty("scripts");
-        includeLiquibaseTables = Boolean.valueOf(getProperties().getProperty("includeLiquibaseTables", "false"));
+        String liquibaseContexts = getProperties().getProperty("contexts", "");
```

Comment:
I don't think this should be prefixed with the `database.` prefix as it is not a database property. But maybe it could be prefixed with something else, currently just went based on the `scripts` property.

Reply:
Inside of Liquibase, the `Contexts` object gets passed along to `changeLogParameters`. I'd say, we call it `changeLogParameters.contexts`.